config/settings.properties

```diff
@@ -1,6 +1,9 @@
 # WASAPI URL endpoint
 baseurl=http://example.org
 
+# WASAPI authentication URL
+authurl=http://example.org/login
```

Comment:
I put a question to the wasapi slack channel as to whether `/login` should be a declared path in wasapi;  if so, we would be able to remove this from the settings file.  (But PR doesn't have to wait for that decision ... we can remove it later).